ntc_templates/templates/cisco_asa_show_interface_ip_brief.textfsm

```diff
@@ -0,0 +1,8 @@
+Value INTF (\S+)
+Value IPADDR (\S+)
+Value STATUS (up|down|administratively down)
+Value PROTO (up|down)
+
+Start
+  ^${INTF}\s+${IPADDR}\s+\w+\s+\w+\s+${STATUS}\s+${PROTO} -> Record
+
```

Comment:
Can you add:

```
  ^\s*$$
  ^. -> Error
```

To all of the templates as well.

Reply:
all done